playbooks/roles/stunnel/tasks/main.yml

```diff
@@ -40,6 +40,7 @@
   systemd:
     name: stunnel4.service
     state: stopped
+    enabled: no
```

Comment:
I'm not a systemd guru; I just looked this up. You might want to throw in `masked: yes`, since this package should *never* should be started, regardless of whether other packages depend on it. https://askubuntu.com/a/816378

Reply:
that sounds reasonable, done!